examples/data_pipeline/transforms.py

```diff
@@ -87,16 +87,27 @@ def __init__(self, vocab):
         super(PyTextVocabTransform, self).__init__()
         self.vocab = vocab
 
+    def forward(self, tokens):
+        return [self.vocab.idx[token] if token in self.vocab.idx.keys() else 0 for token in tokens]
+
+
+class PyTextScriptVocabTransform(nn.Module):
+    r"""Vocab transform
+    """
+
+    def __init__(self, vocab):
+        super(PyTextScriptVocabTransform, self).__init__()
+        self.vocab = vocab
+
     def forward(self, tokens: List[str]) -> List[int]:
```

Comment:
Why not `List[List[str]]`?

Reply:
I will update this to use the batch format